src/ResourceManager/Sql/Commands.Sql/VulnerabilityAssessment/Cmdlet/VulnerabilityAssessmentRuleBaseline/SqlDatabaseVulnerabilityAssessmentRuleBaselineCmdletBase.cs

```diff
@@ -0,0 +1,54 @@
+// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
+using Microsoft.Azure.Commands.Sql.VulnerabilityAssessment.Model;
+using Microsoft.Azure.Commands.Sql.VulnerabilityAssessment.Services;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Sql.VulnerabilityAssessment.Cmdlet.VulnerabilityAssessmentRuleBaseline
+{
+    /// <summary>
+    /// The base class for all Azure Sql Database Vulnerability Assessment Rule Baseline Management Cmdlets
+    /// </summary>
+    public abstract class SqlDatabaseVulnerabilityAssessmentRuleBaselineCmdletBase : SqlDatabaseVulnerabilityAssessmentOperationCmdletBase<DatabaseVulnerabilityAssessmentRuleBaselineModel, SqlVulnerabilityAssessmentRuleBaselineAdapter>
+    {
+        /// <summary>
+        /// Gets or sets the rule ID which identifies the rule to set the baseline results to.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The rule ID which identifies the rule to set the baseline results to.")]
+        [ValidateNotNullOrEmpty]
+        public string RuleId { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the baseline results should apply on a server level rule identified by the RuleId
+        /// </summary>
+        [Parameter(Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Specifies whether the baseline results should apply on a server level rule identified by the RuleId")]
+        public SwitchParameter RuleAppliesToMaster { get; set; }
```

Comment:
`Get-Rule -RuleAppliesToMaster` - isn't "Rule" redundant here? Seems to me it should just be `Get-Rule -Master`

Reply:
The cmdlet is Get-RuleBaseline. If I use -Master it may reference a master baseline which doesn't exist